<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shankar-pc" name="Test" tests="14" failures="12" timestamp="4 Aug 2015 06:04:21 GMT" time="551.938" errors="0">
  <testcase name="testForgotPassword" time="22.421" classname="com.mrosupply.automation.ForgotPasswordTest"/>
  <testcase name="testRegistrationProcess" time="27.837" classname="com.mrosupply.automation.RegistrationProcessTest">
    <failure type="java.lang.AssertionError" message="2nd step - Text has not been entered into FirstName text field">
      <![CDATA[java.lang.AssertionError: 2nd step - Text has not been entered into FirstName text field
	at com.mrosupply.automation.RegistrationProcessTest.testRegistrationProcess(RegistrationProcessTest.java:122)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testRegistrationProcess -->
  <testcase name="testDeleteItemFromShoppingCartTest" time="31.216" classname="com.mrosupply.automation.DeleteItemFromShoppingCartTest">
    <failure type="java.lang.AssertionError" message="Caught Exception expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception expected [true] but found [false]
	at com.mrosupply.automation.DeleteItemFromShoppingCartTest.testDeleteItemFromShoppingCartTest(DeleteItemFromShoppingCartTest.java:97)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteItemFromShoppingCartTest -->
  <testcase name="testAddItemToShoppingCartTest" time="6.356" classname="com.mrosupply.automation.AddItemToShoppingCartTest">
    <failure type="java.lang.AssertionError" message="Caught Exception expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception expected [true] but found [false]
	at com.mrosupply.automation.AddItemToShoppingCartTest.testAddItemToShoppingCartTest(AddItemToShoppingCartTest.java:102)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddItemToShoppingCartTest -->
  <testcase name="testNoDuplicateItemInShoppingCart" time="289.456" classname="com.mrosupply.automation.NoDuplicateItemInShoppingCartTest">
    <failure type="java.lang.AssertionError" message="Caught Exception expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception expected [true] but found [false]
	at com.mrosupply.automation.NoDuplicateItemInShoppingCartTest.testNoDuplicateItemInShoppingCart(NoDuplicateItemInShoppingCartTest.java:101)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testNoDuplicateItemInShoppingCart -->
  <testcase name="testAddQtyToCart" time="36.984" classname="com.mrosupply.automation.AddQtyToCart">
    <failure type="java.lang.AssertionError" message="Caught Exception
 null expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception
 null expected [true] but found [false]
	at com.mrosupply.automation.AddQtyToCart.testAddQtyToCart(AddQtyToCart.java:188)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddQtyToCart -->
  <testcase name="testAddQtyToCart" time="34.711" classname="com.mrosupply.automation.AddQtyToCart">
    <failure type="java.lang.AssertionError" message="Caught Exception
 null expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception
 null expected [true] but found [false]
	at com.mrosupply.automation.AddQtyToCart.testAddQtyToCart(AddQtyToCart.java:188)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddQtyToCart -->
  <testcase name="testAddQtyToCart" time="7.266" classname="com.mrosupply.automation.AddQtyToCart">
    <failure type="java.lang.AssertionError" message="Caught Exception
 null expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception
 null expected [true] but found [false]
	at com.mrosupply.automation.AddQtyToCart.testAddQtyToCart(AddQtyToCart.java:188)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddQtyToCart -->
  <testcase name="testAddQtyToCart" time="23.258" classname="com.mrosupply.automation.AddQtyToCart">
    <failure type="java.lang.AssertionError" message="Caught Exception
 null expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception
 null expected [true] but found [false]
	at com.mrosupply.automation.AddQtyToCart.testAddQtyToCart(AddQtyToCart.java:188)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddQtyToCart -->
  <testcase name="testAddQtyToCart" time="8.808" classname="com.mrosupply.automation.AddQtyToCart">
    <failure type="java.lang.AssertionError" message="Caught Exception
 null expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception
 null expected [true] but found [false]
	at com.mrosupply.automation.AddQtyToCart.testAddQtyToCart(AddQtyToCart.java:188)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testAddQtyToCart -->
  <testcase name="testIncrementsOfQuantityOnly_1" time="25.951" classname="com.mrosupply.automation.IncrementsOfQuantityOnly_1_Test">
    <failure type="java.lang.AssertionError" message="Text has not been successfully added into qty_search field">
      <![CDATA[java.lang.AssertionError: Text has not been successfully added into qty_search field
	at com.mrosupply.automation.IncrementsOfQuantityOnly_1_Test.testIncrementsOfQuantityOnly_1(IncrementsOfQuantityOnly_1_Test.java:80)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testIncrementsOfQuantityOnly_1 -->
  <testcase name="testUnableToAddObsoleteItemInTheCart" time="5.9" classname="com.mrosupply.automation.UnableToAddObsoleteItemInTheCartTest"/>
  <testcase name="testItemsInCartFromProductDetailsPage" time="8.373" classname="com.mrosupply.automation.ItemsInCartFromProductDetailsPageTest">
    <failure type="java.lang.AssertionError" message="Caught Exception expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Caught Exception expected [true] but found [false]
	at com.mrosupply.automation.ItemsInCartFromProductDetailsPageTest.testItemsInCartFromProductDetailsPage(ItemsInCartFromProductDetailsPageTest.java:118)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testItemsInCartFromProductDetailsPage -->
  <testcase name="incrementsOfQuantityOnlyTest" time="23.195" classname="com.mrosupply.automation.IncrementsOfQuantityOnlyTest">
    <failure type="java.lang.AssertionError" message="Qty has not been changed successfully">
      <![CDATA[java.lang.AssertionError: Qty has not been changed successfully
	at com.mrosupply.automation.IncrementsOfQuantityOnlyTest.incrementsOfQuantityOnlyTest(IncrementsOfQuantityOnlyTest.java:83)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- incrementsOfQuantityOnlyTest -->
</testsuite> <!-- Test -->
